src/LinAlg/CMakeLists.txt

```diff
@@ -19,16 +22,7 @@ if(HIOP_SPARSE)
     endif(HIOP_USE_STRUMPACK)
 endif()
 
-if(HIOP_USE_RAJA)
-  set(hiopLinAlg_SRC ${hiopLinAlg_SRC} hiopLinSolverIndefDenseMagma.cpp)
-  foreach(f hiopVectorRajaPar.cpp
-            hiopVectorIntRaja.cpp
-            hiopMatrixRajaDense.cpp
-            hiopMatrixRajaSparseTriplet.cpp)
-    set_source_files_properties(${f} PROPERTIES LANGUAGE CUDA)
-  endforeach()
-endif()
-
+# Add RAJA/Umpire sources when enabled
 if(HIOP_USE_UMPIRE)
   set(hiopLinAlg_SRC ${hiopLinAlg_SRC}
     hiopVectorRajaPar.cpp
```

Comment:
Since now Raja is required whenever HIOP_USE_UMPIRE=ON, should we use change the if condition to
if(HIOP_USE_UMPIRE AND HIOP_USE_RAJA). I think this will also address the issue in #121

Reply:
@nychiang, RAJA has been implicitly required to be on whenever Umpire was on and we failed to make proper enforcements in the CMake code. I think the simpler solution would be to use only one of `HIOP_USE_UMPIRE` or `HIOP_USE_RAJA` and once it is set to `ON` enable both. I can't think of a use case where you would use one without the other in HiOp. I would suggest refactoring CMake and addressing #121 shortly after this and #120 PRs are merged.